integration-tests/README.md

```diff
@@ -77,5 +77,19 @@ node --inspect-brk node_modules/.bin/jest [test name] --runInBand
 
 Using your IDE you can connect to the debug port and set breakpoints. More info [here](https://jestjs.io/docs/en/troubleshooting).
 
+#### Detecting Flakiness
+You can run: 
+```
+yarn test-flakes [test name]
+```
+By default this will run your test 100 times and display the stack trace of every failure encountered.
+
+You can tweak this with the following settings:
+
+Setting | Default | Description
+--------|-------:|------------|
+RUNS | 100 | The number of test runs you want to execute (Default: 100) 
+CONCURRENCY | 10 | The size of the browser pool / the number of simultaneous tests to run.
+CLUSTER_TIMEOUT_MINUTES | 120 | The number of minutes before the overall test times out. 120 is overkill. If you want to run hundreds over iterations, it is best to overestimate this value
```

Comment:
Thanks for the documentation! Any guidance on what may be the limit on `CONCURRENCY` on a Mac?